### PR TITLE
Heartbeat for long running activity

### PR DESCRIPTION
**Pull request by benmoriceau**
## What
Add an heartbeat for tghe long running activities

### CODE REVIEW

File: airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/shared/ActivityConfiguration.java [hunk 1]
```diff
@@ -26,4 +26,11 @@
       .setRetryOptions(TemporalUtils.NO_RETRY)
       .build();
 
+  public static final ActivityOptions LONG_RUN_OPTIONS = ActivityOptions.newBuilder()
+      .setScheduleToCloseTimeout(Duration.ofDays(MAX_SYNC_TIMEOUT_DAYS))
```
Comment (jrhizor) [inline airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/shared/ActivityConfiguration.java:30]: I think we want this to be start to close, not schedule to close.

**Review by jrhizor — APPROVED** [review]
Looks good. Here or in a separate PR, can we make `SHORT_RUN_OPTIONS` with a start to close timeout of like 5 min for other activities?

**Comment by benmoriceau** [comment]
> Looks good. Here or in a separate PR, can we make `SHORT_RUN_OPTIONS` with a start to close timeout of like 5 min for other activities?

I was think about doing that in another PR in order to limit the change that will be released.

File: airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/shared/ActivityConfiguration.java (hunk 1 above)
Comment (benmoriceau) [reply airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/shared/ActivityConfiguration.java:30]: It s propagated to all the other arguments. Will make it explicit.

**Pull request merged**